src/traces/heatmap/hover.js

```diff
@@ -99,6 +104,16 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode, contour)
         text = cd0.text[ny][nx];
     }
 
+    var zLabel;
+    var dummyAx = { // dummy axis for formatting the z value
```

Comment:
Nicely done.

Creating the dummy axis here isn't optimal, but shouldn't make much of a difference. :+1: